packages/cli/commands/config/list.js

```diff
@@ -0,0 +1,63 @@
+const { logger } = require('@hubspot/cli-lib/logger');
+const { getConfig } = require('@hubspot/cli-lib/lib/config');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cli-lib');
+
+const {
+  addConfigOptions,
+  addAccountOptions,
+  addUseEnvironmentOptions,
+  getAccountId,
+  setLogLevel,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const { validateAccount } = require('../../lib/validation');
+const { getTableContents, getTableHeader } = require('../../lib/table');
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validateAccount(options)))) {
+    process.exit(1);
+  }
+};
+
+exports.command = 'list';
+exports.describe = 'List names of portals defined in config';
+
+exports.handler = async options => {
+  loadAndValidateOptions(options);
+
+  const accountId = getAccountId(options);
+
+  trackCommandUsage('functions-list', {}, accountId);
+
+  const config = getConfig();
+  const portalData = config.portals.reduce((acc, portal) => {
```

Comment:
Instead of `reduce`, it looks like `map` is a better fit.

Reply:
Good call, I was doing something different inside of this where `reduce` was better, but `map` is now better. Will update